tests/lib/rules/no-wait-for-empty-callback.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const rule = require('../../../lib/rules/no-wait-for-empty-callback');
+const RuleTester = require('eslint').RuleTester;
+
+// ------------------------------------------------------------------------------
+// Tests
+// ------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester({ parserOptions: { ecmaVersion: 2018 } });
+
+const ALL_WAIT_METHODS = ['waitFor', 'waitForElementToBeRemoved'];
+
+ruleTester.run('no-wait-for-empty-callback', rule, {
+  valid: [
+    ...ALL_WAIT_METHODS.map(m => ({
+      code: `${m}(() => {
+          screen.getByText(/submit/i)
+        })`,
+      errors: [
+        {
+          messageId: 'noWaitForEmptyCallback',
+        },
+      ],
```

Comment:
Errors block is not necessary for valid tests right?

Reply:
Right! Started with the invalid ones and copy pasted 😅
Will update this PR later this week